Assets/MixedRealityToolkit.SDK/Features/UX/Scripts/Sliders/SliderSounds.cs

```diff
@@ -39,6 +39,22 @@ public class SliderSounds : MonoBehaviour
         [SerializeField]
         private float tickEvery = 0.1f;
 
+        [HideInInspector]
+        public float TickEvery
```

Comment:
Nit: I know that the existing tickEvery doesn't have docs (sorry!) - could you add some doccomments for this (and then also add the same text as docs for tickEvery as well?

i.e. something something something...

/// <summary>
/// If playTickSounds is true, the slider will make a sound each tickEvery interval (in seconds).
/// </summary>